.env.example

```
@@ -1,7 +1,20 @@
 
+# app env one of development/production/testing
+APP_ENV=development
+
+# database informations
 DB_HOST_TOOLS=localhost:3306
 DB_NAME=mdwiki_db
 DB_NAME_NEW=mdwiki_db_new
 TOOL_TOOLSDB_USER=root
 TOOL_TOOLSDB_PASSWORD=root
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The TOOL_TOOLSDB_PASSWORD key should go before the TOOL_TOOLSDB_USER key

(UnorderedKey)